test/images/agnhost/grpc-health-checking/grpc-health-checking.go

```diff
@@ -49,13 +50,19 @@ var (
 	delayUnhealthySec int
 	service           string
 	forceUnhealthy    *bool
+	certFile          string
+	privKeyFile       string
 )
 
 func init() {
 	CmdGrpcHealthChecking.Flags().IntVar(&port, "port", 5000, "Port number.")
 	CmdGrpcHealthChecking.Flags().IntVar(&httpPort, "http-port", 8080, "Port number for the /make-serving and /make-not-serving.")
 	CmdGrpcHealthChecking.Flags().IntVar(&delayUnhealthySec, "delay-unhealthy-sec", -1, "Number of seconds to delay before start reporting NOT_SERVING, negative value indicates never.")
 	CmdGrpcHealthChecking.Flags().StringVar(&service, "service", "", "Service name to register the health check for.")
+	CmdGrpcHealthChecking.Flags().StringVar(&certFile, "tls-cert-file", "",
+		"File containing an x509 certificate for gRPC TLS. (CA cert, if any, concatenated after server cert).")
+	CmdGrpcHealthChecking.Flags().StringVar(&privKeyFile, "tls-private-key-file", "",
+		"File containing an x509 private key matching --tls-cert-file.")
```

Comment:
As these 2 parameters can only be used together we should probably validate that user specifies either both of them or none.

Reply:
Do you mean report an error on startup? We only use them if both are specified indeed

Reply:
Added